modules/indexing/src/main/scala/ch/epfl/bluebrain/nexus/kg/indexing/acls/AclIndexer.scala

```diff
@@ -0,0 +1,59 @@
+package ch.epfl.bluebrain.nexus.kg.indexing.acls
+
+import cats.MonadError
+import ch.epfl.bluebrain.nexus.commons.iam.acls.Event._
+import ch.epfl.bluebrain.nexus.commons.iam.acls.{AccessControl, Event, Path, Permission}
+import ch.epfl.bluebrain.nexus.commons.iam.identity.Identity
+import ch.epfl.bluebrain.nexus.commons.sparql.client.SparqlClient
+import ch.epfl.bluebrain.nexus.kg.core.contexts.Contexts
+
+/**
+  * Acl incremental indexing logic that pushes data into an rdf triple store.
+  *
+  * @param client    the SPARQL client to use for communicating with the rdf triple store
+  * @param contexts  the context operation bundle
+  * @param settings  the indexing settings
+  * @param F         a MonadError typeclass instance for ''F[_]''
+  * @tparam F        the monadic effect type
+  */
+@SuppressWarnings(Array("UnusedMethodParameter"))
+class AclIndexer[F[_]](client: SparqlClient[F], contexts: Contexts[F], settings: AclIndexingSettings)(
```

Comment:
We don't need `contexts` here. This is just used in the other indexers to expand the context which might come in the form of a URI. In ACLs that's not used.